en/editor/extension/install.md

```diff
@@ -1,17 +1,17 @@
 # Installation and sharing
 
-Cocos Creator 3D will search for and automatically open the extensions under the "Global" and "Project" paths during startup.
+**Cocos Creator 3D** will search for and automatically open the extensions under the **Global** and **Project** paths during startup.
```

Comment:
\*\*Global\*\* and \*\*Project\*\* paths ->\`Global\` and \`Project\` paths

Reply:
thank you